### ISSUE TITLE
Cannot access write-only property 'temp'

### ISSUE THREAD

**Issue by liemteksun** [manifold-systems/manifold#273]
First of all, thanks for this project
This is very useful for me, particularly dealing with BigDecimal

However, I encountered an issue using this plug-in

Below code is valid but **sometimes** manifold system plug-in complains _Cannot access write-only property 'temp'_ 

I say **sometimes**, because sometimes if I rename 'temp' to something else, it doesn't complain anymore - sometimes it still complains
Sometimes when I remove the setTemp, it doesn't complain anymore - sometimes it still complains
When the exact same code is copied & paste, sometimes it shows the error, sometimes it doesn't

But in all cases, it compiles correctly

I think it has something to do with 'manifold-props'
I'm sure it is the Manifold-System's plug-in, because when I disable the plug-in, there is no such error

Can you please tell me how I can fix this

Thanks

```
public class Temp3
{
	protected boolean temp;

	public void test()
	{
		if ( temp )
		{
			System.out.println( "test" );
		}
	}

	public void setTemp( boolean temp )
	{
		this.temp = temp;
	}
}

```

**Comment by rsmckinney** [comment]
@liemteksun Hi.  Assuming you are using IntelliJ (or Android Studio), are you saying the issue manifests only in the editor?  If so, what version of IJ are you using and what version of the manifold plugin are you using?  Thanks.

**Comment by rsmckinney** [comment]
Ok, I've reproduced the issue in the IJ editor; this is a bug exclusive to the IJ editor. A fix will be coming soon. Thanks for reporting this!

**Comment by liemteksun** [comment]
Thanks for fast response

IntelliJ Community Edition 2020.3

Plug in: 2020.3.40

Again, great job

thanks

**Comment by rsmckinney** [comment]
Available in release 2021.1.13
